starknet_devnet/blueprints/base.py

```diff
@@ -44,7 +46,17 @@ def extract_positive(request_json, prop_name: str):
     return value
 
 
-def extract_hex_string(request_json, prop_name: str) -> int:
+def to_int(value) -> int:
```

Comment:
The output type can be inferred (but it's good practice to have it stated explicitly). The input parameter type is the one that's more important to be annotated (`str` in this case).

Reply:
Fixed in https://github.com/Shard-Labs/starknet-devnet/commit/87e26c74de56dc31fe8525e410937c189a9d0c28

`custom_int` will be used

Reply:
Fixed in https://github.com/Shard-Labs/starknet-devnet/commit/87e26c74de56dc31fe8525e410937c189a9d0c28

`custom_int` will be used